spec/system/planning_applications/assessment_against_legislation_spec.rb

```diff
@@ -161,11 +161,6 @@
         expect(page).not_to have_content("Previous comments")
       end
 
-      expect(row_with_content("D.1a")).to have_field(
```

Comment:
This is duplicate to https://trello.com/c/aFoNhyjg/1479-fix-flaky-test-for-adding-and-updating-comments